aws-lambda-java-serialization/pom.xml

```diff
@@ -31,47 +32,49 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <relocation.prefix>com.amazonaws.lambda.thirdparty</relocation.prefix>
+        <jackson.version>2.12.6.1</jackson.version>
+        <jackson.jsr10.version>2.12.6</jackson.jsr10.version>
+        <gson.version>2.8.9</gson.version>
+        <junit.version>5.8.2</junit.version>
     </properties>
 
-    <!-- If you're adding a new dependency, make sure it and its dependencies are relocated in the shade stage below -->
     <dependencies>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.10.0</version>
+            <version>${jackson.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.google.code.gson</groupId>
-            <artifactId>gson</artifactId>
-            <version>2.8.5</version>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+            <version>${jackson.jsr10.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.json</groupId>
-            <artifactId>json</artifactId>
-            <version>20160810</version>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
Why GSON and Jackson ?

Reply:
`GSON` is still used by the [RIC](https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-runtime-interface-client/src/main/java/com/amazonaws/services/lambda/runtime/api/client/EventHandlerLoader.java#L82) during serialization when the client context platform is Android

Reply:
Why not migrate Android to Jackson ?

Reply:
It would be a breaking change for customers that are using the Java runtime for developing Android applications because the Jackson's serialization for Optionals, Date, LocalDate and Calendar types is different compared to the Gson one.

However, we are exploring this opportunity.

Reply:
That's why v4 is a Major Version. Assumes there's breaking changes.